pbj-integration-tests/src/test/java/com/hedera/pbj/intergration/test/Issue96Test.java

```diff
@@ -0,0 +1,281 @@
+package com.hedera.pbj.intergration.test;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.hedera.pbj.runtime.ProtoParserTools;
+import com.hedera.pbj.runtime.io.buffer.BufferedData;
+import com.hedera.pbj.runtime.io.buffer.Bytes;
+import com.hedera.pbj.test.proto.pbj.Everything;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.HexFormat;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+final class Issue96Test {
+
+    // ================================================================================================================
+    // Verify common comments.
+    @Test
+    @DisplayName("Issue 96")
+    void issue96() {
+        final var unhexed = HexFormat.of().parseHex(testData);
+        assertThrows(InvalidProtocolBufferException.class, () ->
+                com.hederahashgraph.api.proto.java.ServicesConfigurationList.parseFrom(unhexed));
+        assertThrows(Exception.class, () ->
+                com.hedera.hapi.node.base.ServicesConfigurationList.PROTOBUF.parseStrict(
+                        BufferedData.wrap(unhexed)));
+
+    }
+
```

Comment:
I wonder if it is worth making ALL these tests parameterized with a integer passed in for truncationLength with values of `[-1,-2,-3,-4,-5,-6,-7,-8]` Just to make sure a random truncation will throw exception.

Reply:
Added parameterized tests.